tests/pulser_simulation/test_qutip_backend_v2.py

```diff
@@ -22,13 +22,26 @@
 
 import pulser
 from pulser.backend.default_observables import Energy, Occupation, StateResult
+from pulser.backend.observable import Callback
 from pulser_simulation.qutip_backend import QutipBackendV2
 from pulser_simulation.qutip_config import QutipConfig
 from pulser_simulation.qutip_state import QutipState
 from pulser_simulation.simconfig import SimConfig
 from pulser_simulation.simulation import QutipEmulator
 
 
+class CountCalls(Callback):
+    def __init__(self):
+        """
+        Simple callback that counts how often it's been called.
+        The count can be queried after the simulation from self.counter.
+        """
```

Comment:
I'm afraid this will still be problematic, try this instead
```suggestion
class CountCalls(Callback):
    """Simple callback that counts how often it's been called.
    
    The count can be queried after the simulation from self.counter.
    """
    def __init__(self):
        """Initializes a CountCalls instance."""
```